### PR TITLE
Update docs for IUF and remove deprecated procedures

### PR DESCRIPTION
**Pull request by keopp-hpe**
#### Summary and Scope

This PR updates some of the UAN docs to reflect the use of IUF in install/upgrade and how to use it for an image update.  Some UAN specific data was added to the reference to the IUF procedure on updating images.

The procedure on customizing images manually was removed in favor of using IUF.

### CODE REVIEW

**Review by darrenn-jackson — CHANGES_REQUESTED** [review]
In addition to my deduplication comment, we should also remove `Customizing_UAN_Images_Manually.md` from the ditamap files and from `hugo_toc.json`.

**Comment by alanm-hpe** [comment]
Agreed with @darrenn-jackson. In the meantime I'll approve pending those changes.

**Review by alanm-hpe — APPROVED** [review]

File: docs/portal/developer-portal/install/Install_the_UAN_Product_Stream.md [hunk 1]
```diff
@@ -53,28 +78,28 @@ Input files for `sat bootprep` are provided in the `hpc-csm-software-recipe` VCS
 
 The following instructions describe how to set the root password for UAN/Application nodes.
```
[the remark this answers is not in the document]
Comment (alanm-hpe) [reply docs/portal/developer-portal/install/Install_the_UAN_Product_Stream.md:79]: For now I'm going to leave it as is.

File: docs/portal/developer-portal/uan_admin_guide.ditamap [hunk 2]
```diff
@@ -32,7 +32,6 @@
     </topichead>
     <topichead>
         <topicmeta><navtitle>Advanced UAN Configuration</navtitle></topicmeta>
-        <topicref href="advanced/Customizing_UAN_Images_Manually.md" format="markdown"/>
```
Comment (darrenn-jackson) [inline docs/portal/developer-portal/uan_admin_guide.ditamap:35]: this also needs to be removed from `uan_install_guide.ditamap`.

**Review by darrenn-jackson — CHANGES_REQUESTED** [review]
one fix to make

File: docs/portal/developer-portal/uan_admin_guide.ditamap (hunk 2 above)
Comment (alanm-hpe) [reply docs/portal/developer-portal/uan_admin_guide.ditamap:35]: I don't see it in uan_install_guide.ditamap

**Review by darrenn-jackson — APPROVED** [review]

**Pull request merged**